File: src/main/java/io/renren/RenrenApplication.java
package io.renren;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.support.SpringBootServletInitializer;

/**
 * spring_boot 启动类
 *
 * @author huchunliang
 */
@SpringBootApplication
public class RenrenApplication extends SpringBootServletInitializer {

    public static void main(String[] args) {
        SpringApplication.run(RenrenApplication.class, args);
    }

    @Override
    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
        return application.sources(RenrenApplication.class);
    }
}

File: src/main/java/io/renren/RenrenApplication.kt
package io.renren

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
 * spring_boot 启动类
 *
 * @author huchunliang
 */
@SpringBootApplication
class RenrenApplication

fun main(args: Array<String>) {
    SpringApplication.run(RenrenApplication::class.java, *args)
}
